test/integration/client/statement_timeout-tests.js

```diff
@@ -0,0 +1,69 @@
+'use strict'
+var helper = require('./test-helper')
+var Client = helper.Client
+
+var suite = new helper.Suite()
+
+var conInfo = helper.config
+
+function getConInfo (override) {
+  var newConInfo = {}
+  Object.keys(conInfo).forEach(function (k) {
```

Comment:
```js
return Object.assign(
  {},
  conInfo,
  override
);
```

Reply:
My bad, I started with the `appname-tests.js` file as a starting point. I've updated this function in both files.